templates/web/base/waste/services.html

```diff
@@ -47,6 +47,11 @@
          [% IF c.cobrand.moniker == 'kingston' AND (slwp_garden_sacks AND unit.garden_sacks) %]
              <input type="submit" value="Order more garden sacks" class="waste-service-descriptor waste-service-link">
          [% ELSE %]
+            [% IF c.cobrand.moniker == 'brent' %]
+                <p>
+                  If you already have an active Garden Waste subscription and are looking to add another service, please contact our customer services team on 020 8937 5050 who will be able to do this for you.
```

Comment:
To get here, you must have a subscription, so the first part of this If isn't needed, can just be "If you are looking to add..."